scripts/utils/helper.js

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
use concept of reader to read the data from model, and builder to construct configs in the UI components